packages/aws-cdk-lib/aws-route53-targets/lib/elastic-beanstalk-environment-target.ts

```diff
@@ -10,8 +10,10 @@ import { RegionInfo } from '../../region-info';
  * Only supports Elastic Beanstalk environments created after 2016 that have a regional endpoint.
  */
 export class ElasticBeanstalkEnvironmentEndpointTarget implements route53.IAliasRecordTarget {
-  constructor(private readonly environmentEndpoint: string) {
-  }
+  constructor(
+    private readonly environmentEndpoint: string,
+    private readonly props: ElasticBeanstalkEnvironmentEndpointTargetProps = { evaluateTargetHealth: false },
```

Comment:
Could you explain why only the target for Beanstalk has false specified as the default value?

Reply:
hi @badmintoncryer , thanks for your review. This change should be reverted in one of previous commits but looks like I missed that. Thanks for catching this!